index.qmd

```diff
@@ -5,26 +5,26 @@ toc: false
-Learn how to instantiate these cloud-based environments on demand and tailor them to your project's specific needs with our step-by-step guides and tutorials.
+## For EO Projects
 
-Learn the details of building efficient on-demand services via [APEx's propagation services](./propagation.md).
-Gain insights into ensuring the continuous availability, enhancement, and upscaling of EO algorithms.
-Delve into detailed instructions on leveraging interoperability standards like openEO and OGC Application Package to onboard algorithms with best-practice interoperability.
-Explore our guidance on enhancing algorithm performance, facilitating intercomparison of scientific accuracy, and implementing cloudification strategies for existing toolboxes.
+For EO projects, APEx provides a wide range of services that support project execution and ensure that results remain accessible even after project completion.
 
+* **Dynamic Instantiation of Software and Tools** <br/>APEx can support your project by offering access to essential IT components. These include project websites, data visualization tools, workspaces, and development environments — all dynamically instantiated for your project to use. Learn more about APEx Instantiation Services and how to customize them to fit your project’s specific requirements.
 
-Navigate through our technical documentation to harness the full potential of APEx services and propel your EO projects to new heights.
+* **Support for Service Development** <br/>During your project, APEx assists in developing efficient on-demand services through its [propagation services](./propagation.md). These services can support your project with maintaining continuous availability, enhancing, and upscaling EO algorithms. Explore detailed instructions on leveraging interoperability standards like openEO and OGC Application Package to develop and onboard algorithms with best-practice interoperability. Additionally, explore how APEx can provide access guidance on improving algorithm performance, facilitating intercomparison exercises for scientific accuracy, and implementing cloudification strategies for existing toolboxes.
+
+## For EO platforms providers
+
+* **Guidelines for ensuring compliance with ESA projects**<br/> APEx provides clear [guidelines](interoperability/algohostingenv.md) that enable platform providers to prepare their systems for hosting and executing EO services developed by ESA projects. By adhering to these guidelines, you can ensure that your platform is aligned with ESA's standards and ready for seamless integration in upcoming projects.
+
+* **Increased acctivity and visibility**<br/> By making your platform compliant with APEx, you can attract more users as it will be promoted as an APEx-compliant platform. This compliance not only enhances your platform's visibility but also makes your offerings more appealing to other projects.
```

Comment:
```suggestion
* **Increased activity and visibility**<br/> By making your platform compliant with APEx, you can attract more users as it will be promoted as an APEx-compliant platform. This compliance not only enhances your platform's visibility but also makes your offerings more appealing to other projects.
```